client/blocks/eligibility-warnings/index.jsx

```diff
@@ -27,45 +27,65 @@ function getHoldMessages( translate ) {
+			supportUrl: 'https://wordpress.com/help'
+		},
 		NO_BUSINESS_PLAN: {
 			title: translate( 'Business plan required' ),
 			description: translate( 'This feature is only allowed on sites with a business plan.' ),
 			supportUrl: 'https://support.wordpress.com/'
 		},
+		NO_JETPACK_SITES: {
+			title: translate( 'Jetpack site' ),
+			description: translate( 'This feature is not supported on Jetpack sites.' ),
+			supportUrl: 'https://wordpress.com/help'
+		},
 		NO_VIP_SITES: {
 			title: translate( 'VIP site' ),
 			description: translate( 'This feature is not supported on VIP sites.' ),
-			supportUrl: 'https://support.wordpress.com/'
+			supportUrl: 'https://wordpress.com/help'
 		},
-		NO_WPCOM_NAMESERVERS: {
-			title: translate( 'No WordPress.com name servers' ),
-			description: translate( 'Your custom domain must point to WordPress.com name servers.' ),
+		SITE_PRIVATE: {
+			title: translate( 'Private site' ),
+			description: translate( 'This feature is not supported on private sites.' ),
 			supportUrl: 'https://support.wordpress.com/'
 		},
+		SITE_GRAYLISTED: {
+			title: translate( 'Flagged site' ),
+			description: translate( 'This feature is not supported on sites that are not in good standing.' ),
+			supportUrl: 'https://wordpress.com/help'
+		},
 		NON_ADMIN_USER: {
 			title: translate( 'Admin access required' ),
 			description: translate( 'Only site administrators are allowed to use this feature.' ),
-			supportUrl: 'https://support.wordpress.com/'
+			supportUrl: 'https://support.wordpress.com/user-roles/'
 		},
 		NOT_USING_CUSTOM_DOMAIN: {
 			title: translate( 'No custom domain' ),
 			description: translate( 'Your site must use a custom domain to use this feature.' ),
-			supportUrl: 'https://support.wordpress.com/'
+			supportUrl: 'https://support.wordpress.com/register-domain/'
 		},
 		NOT_DOMAIN_OWNER: {
 			title: translate( 'Not a custom domain owner' ),
 			description: translate( 'You must be the owner of the primary domain subscription to use this feature.' ),
-			supportUrl: 'https://support.wordpress.com/'
+			supportUrl: 'https://support.wordpress.com/domains/'
 		},
-		SITE_GRAYLISTED: {
-			title: translate( 'Flagged site' ),
-			description: translate( 'This feature is not supported on sites that are not in good standing.' ),
-			supportUrl: 'https://support.wordpress.com/'
+		NO_WPCOM_NAMESERVERS: {
+			title: translate( 'No WordPress.com name servers' ),
+			description: translate( 'Your custom domain must point to WordPress.com name servers.' ),
+			supportUrl: 'https://support.wordpress.com/domain-helper/'
 		},
-		SITE_PRIVATE: {
-			title: translate( 'Private site' ),
-			description: translate( 'This feature is not supported on private sites.' ),
-			supportUrl: 'https://support.wordpress.com/'
+		NOT_RESOLVING_TO_WPCOM: {
+			title: translate( 'Primary domain not pointing to WordPress.com servers' ),
+			description: translate( 'Your primary domain must point to WordPress.com servers.' ),
```

Comment:
Hi! I've found a possible matching string that has already been translated **17** times:
`translate( 'Your custom domain must point to WordPress.com name servers.' )` *ES Score: 11*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).